web/src/features/hfiCalculator/components/WeeklyViewTable.tsx

```diff
@@ -138,9 +150,30 @@ export const WeeklyViewTable = (props: Props): JSX.Element => {
                         key={`zone-${areaName}`}
                         data-testid={`zone-${areaName}`}
                       >
-                        <TableCell className={classes.planningArea} colSpan={5}>
-                          {area.name}
-                        </TableCell>
+                        <StickyCell
+                          left={0}
+                          zIndexOffset={10}
+                          backgroundColor={'#e9ecf5'}
+                          colSpan={2}
+                        >
+                          <TableCell className={classes.noBottomBorder}>
+                            {area.name}
+                          </TableCell>
+                        </StickyCell>
+                        <TableCell
+                          className={`${classes.planningArea} ${classes.unstickyHeaderCell}`}
+                        ></TableCell>
+                        <StickyCell
+                          left={230}
+                          zIndexOffset={10}
+                          backgroundColor={'#e9ecf5'}
```

Comment:
Styling in 2 places here, `backgroundColor` and `className`. Can we add the background color to a new class so we save ourselves maintenance pain in the future?